sympy/stats/tests/test_stochastic_process.py

```diff
@@ -723,7 +723,7 @@ def test_GammaProcess_symbolic():
     # Equivalent to E(2*X(1)) + E(X(1)**2) + E(X(1)**3), where E(X(1)) == g/l
     assert E(X(t)**2 + X(d)*2 + X(y)**3, Contains(t, Interval.Lopen(0, 1))
         & Contains(d, Interval.Lopen(1, 2)) & Contains(y, Interval.Ropen(3, 4))) == \
-            2*g/l + (g**2 + g)/l**2 + (g**3 + 3*g**2 + 2*g)/l**3
+            2*g/l + 2*((g**2/2 + g/2)/l**2) + 6*((g**3/6 + g**2/2 + g/3)/l**3)
```

Comment:
This simplification issue may be valid, and I'm not sure why the changes here start not to simplify this anymore.

Reply:
Possibly the difference is because of the 2-arg `Mul(Rational, Add)` which breaks associativity of multiplication:
https://github.com/sympy/sympy/blob/0ec134d277f14961e8b6629182895fc986658c2e/sympy/core/mul.py#L278-L300